src/ViewModel/Converter/Block/ImageConverter.php

```diff
@@ -27,7 +27,7 @@ public function convert($object, string $viewModel = null, array $context = [])
     {
         $image = $object->getImage();
 
-        $imageViewModel = $this->viewModelConverter->convert($image, null, ['width' => $object->isInline() ? 365 : 538]);
+        $imageViewModel = $this->viewModelConverter->convert($image, null, ['width' => $object->isInline() ? 293 : 610]);
```

Comment:
These are based on article text pages, which are smaller than on article figures and other (eg blog post) pages. Which one should we base it on? Can probably work out what size it should be, but means less browser caching between the two article pages (would then be different sizes). /cc @davidcmoulton

Reply:
If they can't all be exactly the right size, we should optimise for the article text page.